example.js

```diff
@@ -1,35 +1,29 @@
 
+// to run, use
+//   electro example.js {image_file_name}
+// or
+//   electro example.js
+// to use file selector.
 
 var h = require('hyperscript')
 
 var hyperfile = require('hyperfile')
 var hypercrop = require('./')
 
-//var fileinput = hyperfile(function (buffer) {
-//  BUF = buffer
-//  var img = h('img', {src: 
-//      'data:image/jpg;base64,'+buffer.toString('base64')
-//    })
-
-var img = h('img', {src: '/home/dominic/pictures/IMG_20150905_101949.jpg'})
```

Comment:
I was gonna propose this images : https://upload.wikimedia.org/wikipedia/commons/6/6f/Statue_of_Rama_in_Kangra_district_of_Himachal_Pradesh.jpg

![](https://upload.wikimedia.org/wikipedia/commons/6/6f/Statue_of_Rama_in_Kangra_district_of_Himachal_Pradesh.jpg)

it's the Rama (an avatar?)